tox.ini

```diff
@@ -37,25 +30,37 @@ deps =
     readme
 
 [testenv:py27-pytest]
-commands = pytest {[tox]test_dir}
+deps =
+    -r requirements.txt
+    pytest
+    docker
+commands = pytest -v {[tox]test_dir}
 
 [testenv:py35-pytest]
-passenv = TRAVIS,CODACY_PROJECT_TOKEN
+deps =
+    -r requirements.txt
+    pytest
+    pytest-cov
+    coverage
+    codacy-coverage
+    docker
 whitelist_externals = sed
                       bash
 commands =
-    pytest --cov={envsitepackagesdir}/ephemeris --cov-report xml {[tox]test_dir}
+    pytest -v --cov={envsitepackagesdir}/ephemeris --cov-report xml {[tox]test_dir}
     # Replace the installed package directory by the source directory.
     # This is needed for codacy to understand which files have coverage testing
     sed -i 's#{envsitepackagesdir}#src#' coverage.xml
     # If running on travis, send a coverage report to codacy.
     # The CODACY_PROJECT_TOKEN env variable should be defined on travis.
-    bash -c 'if [ "$TRAVIS" = "true" ] ; then  python-codacy-coverage -r coverage.xml; fi'
-
+    # This is only the case when the pull request is run from the ephemeris repo. So this is checked.
+    bash -c 'if [ "$TRAVIS" = "true" ] && [ "$TRAVIS_REPO_SLUG" = "galaxyproject/ephemeris" ]  ; then  python-codacy-coverage -r coverage.xml; fi'
```

Comment:
I think this belongs in the deploy stage of the travis.yml. We shouldn't adapt our testing to travis, we should adapt travis to our testing. Same with the sed command.

Reply:
:+1: will change this.
Also there is some problems with codacy coverage still... Working on it.